lein-cljfmt/src/leiningen/cljfmt.clj

```diff
@@ -59,20 +59,28 @@
      (apply check project (format-paths project))
      (main/abort "No project found and no source paths provided")))
   ([project path & paths]
-   (let [files   (mapcat (partial find-files project) (cons path paths))
-         flag    (atom 0)]
-     (doseq [f     files
-             :let  [original (slurp f)
-                    revised  (reformat-string project original)]
-             :when (not= original revised)]
-       (main/warn (project-path project f) "has incorrect formatting:")
-       (main/warn (format-diff project f original revised))
-       (swap! flag inc))
-     (if (zero? @flag)
+   (let [files             (mapcat (partial find-files project) (cons path paths))
+         incorrect-counter (atom 0)
+         failure-counter   (atom 0)]
+     (doseq [^java.io.File f files]
+       (let [original (slurp f)]
+         (try
+           (let [revised (reformat-string project original)]
+             (when (not= original revised)
+               (main/warn (project-path project f) "has incorrect formatting:")
+               (main/warn (format-diff project f original revised))
+               (swap! incorrect-counter inc)))
+           (catch Exception e
+             (main/warn "Failed to format file:" (.getPath f))
```

Comment:
Instead of `(.getPath f)`, `(project-path project f)` would produce a more concise filepath.

I also think we might want to print out the exception and stacktrace as well. So maybe:

```clojure
(main/warn (project-path project f) "could not be formatted:")
(binding [*out* *err*] (clojure.stacktrace/print-stack-trace e))
```